.changeset/spicy-zebras-dream.md

```diff
@@ -0,0 +1,5 @@
+---
+"@localyze-pluto/theme": minor
+---
+
+[Default Theme] Add decorative01 and decorative02 bg colors
```

Comment:
Is this text something we add ourselves?

Reply:
Yes, it is. 🙏🏻